protocols/kad/src/behaviour.rs

```diff
@@ -441,6 +441,11 @@ where
         &mut self.store
     }
 
+    /// Returns the number of iterative queries that are currently being performed.
+    pub fn num_active_iterative_queries(&self) -> usize {
```

Comment:
```suggestion
    pub fn num_active_queries(&self) -> usize {
```

A query could also be a [`FixedPeersIter`](https://github.com/libp2p/rust-libp2p/blob/master/protocols/kad/src/query/peers/fixed.rs#L28) which only operates on a predefined set of peers.